### PR TITLE
style: added bug issue template so that user can create bug on kubern…

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**
…etes-client/fabric8

Created Issue Template for the repository.

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [ ] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry regarding this change
 - [ ] I have implemented unit tests to cover my changes
 - [x] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+`$ Mention kubernetes-client/fabric8 version from pom.xml file`
+
+<!-- Verify pom.xml file and insert the java-operator-sdk version here. -->
+
+`$ java -version`
+
+<!-- Insert the output of `java -version` here -->
+
+`$ kubectl version`
```
Comment (rohanKanojia) [inline .github/ISSUE_TEMPLATE/bug-report.md:45]:
```suggestion
`Fabric8 Kubernetes Client Version`
```

**Review by oscerd — APPROVED** [review]

**Comment by laxmikantbpandhare** [comment]
@rohanKanojia or @manusa - Could you please take a look at this PR. It is already approved by @oscerd . Thank you.

**Comment by metacosm** [comment]
I don't have an opinion on this, either way.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (rohanKanojia) [reply .github/ISSUE_TEMPLATE/bug-report.md:45]: Asking user to provide kubectl version doesn't make sense in context of Kubernetes Java SDK? Maybe we should remove this section altogether? WDYT?

**Review by oscerd — APPROVED** [review]

**Comment by oscerd** [comment]
I think we should merge this one.

**Comment by rohanKanojia** [comment]
@laxmikantbpandhare : We  (@sunix and @manusa and I) discussed this and decided we would like to have an issue template like [Eclipse Che](https://github.com/eclipse/che/issues/new/choose) .

Would it be possible for you to modify your PR to make it look like che instead. Maybe we just need to do same thing as done [here](https://github.com/eclipse/che/tree/main/.github/ISSUE_TEMPLATE)

**Comment by laxmikantbpandhare** [comment]
@rohanKanojia Sure, I will update the PR soon.

**Comment by laxmikantbpandhare** [comment]
@rohanKanojia - I have not added `release.md` as I am not sure about the process. If that is necessary could you please help me with the process.

**Comment by rohanKanojia** [comment]
I don't think we create specific issues for releases. We can leave the `release.md` out.

File: .github/ISSUE_TEMPLATE/a_question.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    attributes:
+      value: |
+        ### Asking a new Question 🤔
+        🔍  Before opening a new issue please search existing issues at https://github.com/fabric8io/kubernetes-client/issues
+
+        🤔  To make it easier for us to help you, please include as much useful information as possible.
+
+        Useful Links:
+
+          - 📄  Documentation: https://fabric8.io/
```
Comment (rohanKanojia) [inline .github/ISSUE_TEMPLATE/a_question.yml:32]: Shall we replace this with CHEATSHEET URL?

https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md

Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/a_question.yml:32]: sure.

Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/a_question.yml]: Modified.

**Comment by laxmikantbpandhare** [comment]
> I don't think we create specific issues for releases. We can leave the `release.md` out.

Sure. thank you.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Describe the bug
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: kubernetes-client version
+      description: if workspace is running, version can be obtained with help/about menu
```
Comment (rohanKanojia) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]: I'm not sure if it's applicable to a library like fabric8 Kubernetes Client. Maybe we can say something like `Fabric8 Kubernetes Client version (can be found in pom.xml dependency or build.gradle implementation section)`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    attributes:
+      label: Runtime
+      description: What is the kubernetes flavor ?
+      multiple: true
+      options:
+        - Kubernetes (vanilla)
+        - OpenShift
+        - minikube
+        - CodeReady Container
+        - Docker Desktop
```
Comment (rohanKanojia) [inline .github/ISSUE_TEMPLATE/bug_report.yml:97]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+        - GCE
+        - "Dev Sandbox (workspaces.openshift.com)"
+        - other (please specify in additional context)
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Eclipse Che Logs
```
Comment (rohanKanojia) [inline .github/ISSUE_TEMPLATE/bug_report.yml:141]:
```suggestion
      label: Fabric8 Kubernetes Client Logs
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Modified accordingly.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 5 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Modified accordingly.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 4 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Modified accordingly.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+
+  - type: dropdown
```
Comment (rohanKanojia) [inline .github/ISSUE_TEMPLATE/bug_report.yml:107]: Not applicable to Fabric8

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+      description: A clear and concise description of what you expected to happen.
+      placeholder:
+    validations:
+      required: true
+
+  - type: dropdown
+    id: runtime
+    attributes:
+      label: Runtime
+      description: What is the kubernetes flavor ?
```
Comment (rohanKanojia) [inline .github/ISSUE_TEMPLATE/bug_report.yml:90]: Do you think there should be a text field for the Kubernetes version also?

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 6 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml:107]: Modified.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 7 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml:90]: Yes, it is needed. Modified and added.

**Review by rohanKanojia — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/a_question.yml [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Question 🤔
+description: Usage question or discussion about Fabric8 kubernetes-client
+labels: [kind/question]
```
Comment (manusa) [inline .github/ISSUE_TEMPLATE/a_question.yml:19]: We shouldn't have this either.
We need to encourage users to use [Discussions](https://github.com/fabric8io/kubernetes-client/discussions) instead.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+        - "5.9.0"
+        - "5.8.0"
+        - "5.7.0"
+        - "5.6.0"
+        - "5.5.0"
+        - "5.4.0"
+        - "5.3.0"
+        - "5.2.0"
+        - "5.1.0"
+        - "other (please specify in additional context)"
```
Comment (manusa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:62]: This dropdown will become outdated very soon.
We can keep it, but we'll probably need to figure out a way to automate something to make it self-updatable

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 6 above)
Comment (manusa) [reply .github/ISSUE_TEMPLATE/bug_report.yml:115]: how?
Section should be removed

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 10]
```diff
@@ -0,0 +1,161 @@
...
+    attributes:
+      label: Environment
+      multiple: true
+      options:
+        - Windows
+        - Linux
+        - macOS
+        - Amazon
+        - Azure
+        - GCE
```
Comment (manusa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:139]: GKE?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 11]
```diff
@@ -0,0 +1,161 @@
...
+    attributes:
+      label: Installation method
+      multiple: true
+      options:
+        - chectl/latest
+        - chectl/next
+        - OperatorHub
+        - other (please specify in additional context)
+    validations:
+      required: true
```
Comment (manusa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:126]: This whole section is not applicable to Fabric8 client (or at least not relevant)

Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml:126]: Updated.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 10 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml:139]: yup

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 9 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml:62]: @manusa - I am not sure about dynamically passing values to it. We can remove this version section completely if not needed. We referred it from [Eclipse Che.](https://github.com/eclipse/che/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml)

Comment (sunix) [reply .github/ISSUE_TEMPLATE/bug_report.yml:62]: maybe our release scripts could update that yaml file adding the version released.

Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/bug_report.yml:62]: yes, absolutely, that is something we have to look for.

File: .github/ISSUE_TEMPLATE/a_question.yml (hunk 8 above)
Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/a_question.yml:19]: So, need to remove entire Question template right?

Comment (manusa) [reply .github/ISSUE_TEMPLATE/a_question.yml:19]: Yes

Comment (laxmikantbpandhare) [reply .github/ISSUE_TEMPLATE/a_question.yml]: Updated.

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**